eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/EventMeshAdminServer.java

```diff
@@ -0,0 +1,131 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.runtime.boot;
+
+import org.apache.eventmesh.runtime.admin.controller.HttpHandlerManager;
+import org.apache.eventmesh.runtime.configuration.EventMeshHTTPConfiguration;
+
+import io.netty.bootstrap.ServerBootstrap;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelOption;
+import io.netty.channel.ChannelPipeline;
+import io.netty.channel.epoll.EpollServerSocketChannel;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.channel.socket.nio.NioServerSocketChannel;
+import io.netty.handler.codec.http.HttpObjectAggregator;
+import io.netty.handler.codec.http.HttpRequest;
+import io.netty.handler.codec.http.HttpRequestDecoder;
+import io.netty.handler.codec.http.HttpResponseEncoder;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class EventMeshAdminServer extends AbstractHTTPServer {
+
+    private HttpConnectionHandler httpConnectionHandler;
+
+    HttpHandlerManager httpHandlerManager;
+
+    public EventMeshAdminServer(int port, boolean useTLS,
+        EventMeshHTTPConfiguration eventMeshHttpConfiguration, HttpHandlerManager httpHandlerManager) {
+        super(port, useTLS, eventMeshHttpConfiguration);
+        this.httpHandlerManager = httpHandlerManager;
+    }
+
+    @Override
+    public void init() throws Exception {
+        super.init("eventMesh-admin-http");
+        httpConnectionHandler = new HttpConnectionHandler();
+    }
+
+    @Override
+    public void start() throws Exception {
+        final Thread thread = new Thread(() -> {
+            final ServerBootstrap bootstrap = new ServerBootstrap();
+            try {
+                bootstrap.group(this.getBossGroup(), this.getIoGroup())
+                    .channel(useEpoll() ? EpollServerSocketChannel.class : NioServerSocketChannel.class)
+                    .childHandler(new AdminServerInitializer(httpHandlerManager))
+                    .childOption(ChannelOption.SO_KEEPALIVE, Boolean.TRUE);
+
+                log.info("AdminHttpServer[port={}] started.", this.getPort());
+
+                bootstrap.bind(this.getPort())
+                    .channel()
+                    .closeFuture()
+                    .sync();
+            } catch (Exception e) {
+                log.error("AdminHttpServer start error!", e);
+                try {
+                    shutdown();
+                } catch (Exception ex) {
+                    log.error("AdminHttpServer shutdown error!", ex);
+                }
+                System.exit(-1);
+            }
+        }, "EventMesh-http-server");
+        thread.setDaemon(true);
+        thread.start();
+        started.compareAndSet(false, true);
+    }
```

Comment:
L86, better not to edit the startup status boolean of an abstract class (AbstractHTTPServer).